### PR TITLE
Post-transform script for Incubator Good Enough Practices lesson

### PR DESCRIPTION
**Pull request by tobyhodges**

### CODE REVIEW

**Review by zkamvar — COMMENTED** [review]
Looks good to me! I have a couple of suggestions to simplify the code and avoid situations where they change the text before you end up transferring the content.

File: carpentries-incubator/good-enough-practices.R [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+writeLines(dataman, to("episodes/02-data_management.md"))
+## software episode
+software <- readLines(to("episodes/03-software.md"))
+q <- 'I\'m conscious that lots of people would like to see and run the pandemic simulation code we are using to model control measures against COVID-19. To explain the background — I wrote the code \\(thousands of lines on undocumented C\\) 13 plus years ago to model flu pandemics…'
+software <- sub(paste0('[\\]{2}["]', q, '[\\]{2}["]'), paste0("'", q, "'"), software)
+writeLines(software, to("episodes/03-software.md"))
+
+# escape dollar signs to avoid USD amounts being mistaken for LaTeX syntax
+track_changes <- readLines(to("episodes/06-track_changes.md"))
+track_changes <- gsub('\\$5', '\\\\\\\\$5', track_changes)
```
[the remark this answers is not in the document]
Comment (zkamvar) [reply carpentries-incubator/good-enough-practices.R:53]: reference for raw strings: https://mpopov.com/blog/2020/05/22/strings-in-r-4.x/

[the remark this answers is not in the document]
Comment (tobyhodges) [reply carpentries-incubator/good-enough-practices.R:53]: Thanks for the raw strings; did not know about that until now. 

> What you wrote will produce:
>
>    I owe you \\$5,
>
> which will produce an extra slash in the rendered HTML

That's what I thought too (specifically, "Surely I cannot possible need _eight_ backslashes" is what I thought) but it really does seem to be necessary.

With `gsub(r'{\$5}', r'{\\$5}', track_changes)`:

rendered result with two backslashes

and with `gsub(r'{\$5}', r'{\\\\$5}', track_changes)`:

rendered result with four backslashes

[the remark this answers is not in the document]
Comment (zkamvar) [reply carpentries-incubator/good-enough-practices.R]: Oof. Good point

**Comment by zkamvar** [comment]
I'm happy to merge this so that you can work with it on the main branch.

**Pull request merged**